app/src/main/java/com/neu/mobileapplicationdevelopment202430/fragment/TeamScreen.kt

```diff
@@ -0,0 +1,104 @@
+package com.neu.mobileapplicationdevelopment202430.fragment
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material3.Button
+import androidx.compose.material3.Surface
+import androidx.compose.material3.Text
+import androidx.compose.runtime.*
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.lifecycle.viewmodel.compose.viewModel
+import androidx.navigation.NavController
+import com.neu.mobileapplicationdevelopment202430.R
+import com.neu.mobileapplicationdevelopment202430.pokemon.TeamViewModel
+import com.neu.mobileapplicationdevelopment202430.pokemon.TeamViewModelFactory
+import com.neu.mobileapplicationdevelopment202430.room.PokemonDatabase
+import com.neu.mobileapplicationdevelopment202430.room.TeamPokemonEntity
+import com.neu.mobileapplicationdevelopment202430.team.TeamPlaceholderCard
+import com.neu.mobileapplicationdevelopment202430.team.TeamPokemonCard
+
+@Composable
+fun TeamScreen(navController: NavController) {
+    val context = LocalContext.current
+    val db = PokemonDatabase.getDatabase(context)
+    val dao = db.teamPokemonDao()
+    val factory = TeamViewModelFactory(dao)
+    val viewModel: TeamViewModel = viewModel(factory = factory)
+    val team by viewModel.team.collectAsState(initial = emptyList())
+
+    // TODO: Remove
```

Comment:
this is fine to keep for now i guess, i'll remove it in my next PR once i've implemented adding pokemon from the random generator.